package.json

```
@@ -13,7 +13,8 @@
     "gulp": "node --max_old_space_size=8048 node_modules/gulp/bin/gulp.js",
     "build": "npm run gulp -- build",
     "test": "npm run gulp -- test",
-    "pack": "npm run gulp -- npm.pack"
+    "pack": "npm run gulp -- npm.pack",
+    "update-meta": "dx-tools update-meta"
```

I suggest to use the same script name in all the repos:

```suggestion
    "update-integration-meta": "dx-tools update-meta"
```